### ISSUE TITLE
Dependencies on Applications shows full JSON data at "latest"

### ISSUE THREAD

**Issue by 78wesley** [FlowFuse/flowfuse#4587]
### Current Behavior

When I open a application and go to the dependencies tab I see that the "Latest" shows the full JSON data.

![image](https://github.com/user-attachments/assets/2872581f-900a-4657-b7e0-b07243ea44c4)

### Expected Behavior

Only show the "Latest" version instead of the full JSON data.

### Steps To Reproduce

Open a application go the dependencies tab.

### Environment

- FlowFuse version: ~
- Node.js version: ~
- npm version: ~
- Platform/OS: ~
- Browser: Brave Version 1.70.119 Chromium: 129.0.6668.70 (Official Build) (64-bit)

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

**Comment by 78wesley** [comment]
Just logged back in and I see now that it has been fixed. I guess it has been fixed by #4589.

**Comment by joepavitt** [comment]
Yep, thanks @78wesley - this has now been resolved by https://github.com/FlowFuse/flowfuse/pull/4589
